pkg/wallet/service/eth/watchsrv/tx_creator_payment.go

```diff
@@ -46,7 +46,7 @@ func (t *TxCreate) CreatePaymentTx() (string, string, error) {
 		return "", "", errors.Wrap(err, "fail to call addrRepo.GetAll(account.AccountTypeClient)")
 	}
 	if t.validateAmount(senderAddr, totalAmount) != nil {
-		return "", "", err
+		return "", "", nil
```

Comment:
It would be better that error from validateAmount() is returned here. 
```suggestion
if err := t.validateAmount(senderAddr, totalAmount); err != nil {
		return "", "", err
```